src/Microsoft.ML.SamplesUtils/SamplesDatasetUtils.cs

```diff
@@ -86,6 +86,38 @@ public static string DownloadSentimentDataset()
         public static string DownloadAdultDataset()
             => Download("https://raw.githubusercontent.com/dotnet/machinelearning/244a8c2ac832657af282aa312d568211698790aa/test/data/adult.train", "adult.txt");
 
+        public static IDataView LoadAdultDataset(MLContext mlContext)
```

Comment:
>LoadAdultDataset [](start = 32, length = 16)

I prefere Shahab approach [https://github.com/dotnet/machinelearning/pull/2517/files#diff-eb95ea0c54ebcf8d695d8d73d5849b0c](https://github.com/dotnet/machinelearning/pull/2517/files#diff-eb95ea0c54ebcf8d695d8d73d5849b0c)

Reply:
I actually took it from his PR...:)
He then changed. I am updating.

---
In reply to: [256547096](https://github.com/dotnet/machinelearning/pull/2476#discussion_r256547096) [](ancestors = 256547096)